aten/src/ATen/ParallelNativeTBB.cpp

```diff
@@ -0,0 +1,75 @@
+#if AT_PARALLEL_NATIVE_TBB
+#include <ATen/Parallel.h>
+
+#include <atomic>
+
+#include "tbb/tbb.h"
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#ifdef TH_BLAS_MKL
+#include <mkl.h>
+#endif
+
+namespace at {
+
+namespace internal {
+  tbb::task_scheduler_init tbb_init(
+      tbb::task_scheduler_init::deferred);
+  tbb::task_arena arena;
```

Comment:
@colesbury - do you remember what the issue / if there was an issue with the approach of having a global arena for tbb control (to set the correct number of threads)?

Reply:
No, I don't remember much other than something like having to set the number of threads before the first TBB launch

Reply:
This arena setup should get rid of that. However, maybe the access needs to be made atomic @ilia-cher?

Reply:
as discussed offline I think I'll have to remove arena from here for perf reason